scss/mixins/_accessibility.scss

```diff
@@ -0,0 +1,14 @@
+//-----------------------------------
+//
+// Accessibility mixins
+// ====================
+//
+//-----------------------------------
+
+// Assert that the parent selector matches one of the selectors passed into $selectors
+@mixin assert-selector($selectors...) {
+  &:not(#{$selectors}) {
+    /*! You must ensure the element matches one of these selectors: #{$selectors} */
+    outline: 2px solid red !important;
```

Comment:
Have you considered also throwing a warning on compilation like:

```
@warn "You must ensure the element matches one of these selectors: #{$selectors}";
```

This will spit back the warning and the line number/file.